docs/themes/oso-tailwind/assets/js/nav.js

```diff
@@ -26,13 +26,15 @@ if (sideBarButton) {
   sideBarButton.addEventListener('click', toggleSideBar);
 }
 
-const langButton = document.getElementById('language-selector-toggle');
-const langsideBar = document.getElementById('language-selector-content');
-if (langButton) {
-  langButton.addEventListener('click', () =>
-    langsideBar.classList.toggle('hidden')
-  );
-}
+window.addEventListener('load', () => {
```

Comment:
Why is this changed to on load? 

`langButton` needs to be declared for the closing logic below (line 53).

Reply:
Oh might be able to avoid this, it wasn't attaching the event listener but I think that was for a different reason